docs/products/kafka/kafka-mirrormaker/howto/log-analysis-offset-sync-tool.md

```diff
@@ -0,0 +1,171 @@
+  </tr>
+  <tr>
+    <td>1</td>
+    <td>Successfully synced, operating normally.</td>
+    <td>Data and offsets are successfully migrated. Consumers consume from the synced
+        offsets with minimal redelivery.</td>
+    <td>No action required</td>
+    <td>Not required</td>
+    <td>Not required</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>2</td>
+    <td>Intentionally not synced because the source partition is empty. Other partitions
+        in this group are synced.</td>
+    <td>The source partition has no data, so MirrorMaker 2 skips syncing it. Consumers
+        start from the beginning when new data arrives.</td>
+    <td>No action required</td>
+    <td>Not required</td>
+    <td>Not required</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>3</td>
+    <td>Intentionally not synced because the source partition is empty. Other partitions
+        in this group are not synced.</td>
+    <td>The source partition has no data, so MirrorMaker 2 skips syncing it. Consumers
+        start from the beginning when new data arrives.</td>
+    <td>No action required</td>
+    <td>Not required</td>
+    <td>Not required</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>4</td>
+    <td>Intentionally not synced because the source group is too old. Other partitions
+        in this group are synced.</td>
+    <td>The source partition has newer data than the committed offset for the consumer
+        group. Consumers will start from the beginning when new data arrives.</td>
+    <td>No action required</td>
+    <td>Not required</td>
+    <td>Not required</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>5</td>
+    <td>Intentionally not synced because the source group is too old. Other partitions
+        in this group are not synced.</td>
+    <td>The source partition has newer data than the committed offset for the consumer
+        group. Consumers will start from the beginning when new data arrives.</td>
+    <td>No action required</td>
+    <td>Not required</td>
+    <td>Not required</td>
+    <td>No</td>
+  </tr>
+  <tr>
+    <td>6</td>
+    <td>Erroneously not synced. Source partition has data, and other partitions for this
+        group are synced.</td>
+    <td>There is an offset syncing issue. Consumers start from the beginning
+        when new data arrives.</td>
+    <td><a href="/docs/platform/howto/support">Create a support ticket</a> or
+        email <a href="mailto:support@aiven.io">Aiven Support</a>.</td>
+    <td>
+      <ul>
+        <li>Use <code>kafka-get-offsets</code> to compare the earliest and latest
+            offsets in the source and target topics.</li>
+        <li>Check the source Kafka dashboard for throughput on the partition.</li>
+        <li>Use <code>kafka-console-consumer</code> to inspect the offset syncs topic
+            and grep for the topic-partition key.</li>
+        <li>Enable TRACE logging for OffsetSyncStore in MirrorMaker 2 to review
+            translation activity.</li>
```

Comment:
Is this something that the customer can do? Is this appropriate for this customer facing document?

Reply:
I am not sure about this. @MichaelTansiniAiven or @jjaakola-aiven, could you confirm? I added it here as it was in the shared Google Doc.